ui/raidboss/data/06-ew/trial/sophia-un.ts

```diff
@@ -111,7 +111,7 @@ const triggerSet: TriggerSet<Data> = {
           en: 'Avoid Dash Attack',
           de: 'Ansturm-Angriff ausweichen',
           fr: 'Évitez l\'attaque Charge',
-          ja: '突進に避け',
+          ja: '突進回避',
```

Comment:
If you are going to change this, can you change sophia-ex.ts instead and run `ts-node util/sync_files.ts` so that these two files stay in sync?